extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -14,5 +14,12 @@
  * limitations under the License.
  */
 
- amp-sticky-ad {
- }
+amp-sticky-ad {
+  position: fixed !important;
+  text-align:center;
+  bottom: 0 !important;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+  max-height: 100px !important;
```

Comment:
@dvoytenko lost your comment on this on github

@zhouyx lets make a quick js bin with a div having 1/6 vh and compute its height (on orientation change) and then recored the findings in a table and then finalize this. 

I want this to be one value and not complicate this logic if the numbers are not very different.